### PR TITLE
Bump gcc version to 11

### PR DESCRIPTION
**Pull request by xThaid**
Debian packages are ready to use in my home directory on the server (`/home/jurb/`).

#999 is required to compile Mimiker under the new GCC version.

### CODE REVIEW

File: toolchain/gnu/config.mk [hunk 1]
```diff
@@ -17,7 +17,7 @@ GMP-URL = "https://gmplib.org/download/gmp/$(GMP).tar.xz"
 MPC-URL = "https://ftp.gnu.org/gnu/mpc/$(MPC).tar.gz"
 CLOOG-URL = "http://www.bastoul.net/cloog/pages/download/$(CLOOG).tar.gz"
 BINUTILS-URL = "https://ftp.gnu.org/gnu/binutils/$(BINUTILS).tar.xz"
-GCC-URL = "https://ftp.gnu.org/gnu/gcc/$(GCC)/$(GCC).tar.xz"
+GCC-URL = "ftp://ftp.fu-berlin.de/unix/languages/gcc/snapshots/11-20210314/$(GCC).tar.xz"
```
Comment (MichalBlk) [inline toolchain/gnu/config.mk:20]: Maybe add a comment which prompts to update the URL after an appropriate release.

**Review by MichalBlk — APPROVED** [review]
LGTM

**Review by pj1031999 — APPROVED** [review]

**Comment by MichalBlk** [comment]
One more thing - shouldn't we change ` /mimiker/toolchain/gnu/debian/changelog`?

**Comment by cahirwpz** [comment]
@MichalBlk Yes, `changelog` file should be updated as well. I'll do it in another PR, since I have to update Docker image as well.

**Pull request merged**